EIPS/eip-2612.md

```diff
@@ -0,0 +1,191 @@
+---
+eip: 2612
+title: `permit`: 712-signed approvals
+author: Martin Lundfall (@Mrchico)
+discussions-to: https://github.com/ethereum/EIPs/pull/2612
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-04-13
+requires: ERC20, ERC712
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A function `permit` extending ERC20 which allows for approvals to be made via `secp256k1` signatures. This kind "account abstraction for ERC20" brings about two main benefits:
```

Comment:
Please use `ERC-20` throughout and make sure the first use is a link to https://eips.ethereum.org/EIPS/eip-20